easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.2.1'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('ncurses', '5.9')]
```

Comment:
I haven't seen a PR for it, and if we go with ncurses 6.0, we need to do it everywhere (e.g. also in libreadline, etc.).